### ISSUE TITLE
Fix SEO links

### ISSUE THREAD

**Issue by jema28** [yalla-coop/website-redesign#98]
Checked the live site at https://www.yallacooperative.com/ but there are 404 links underneath. @yosefanajjar @thejoefriel can we look into this asap? 

<img width="653" alt="Screenshot 2020-11-03 at 06 30 11" src="https://user-images.githubusercontent.com/25960351/97955430-0f14ed00-1d9e-11eb-94ea-2c8f99d0c341.png">

I think I should design a simple 404 page. I'll add an issue for this.

**Comment by jema28** [comment]
404 issue: #99

**Comment by yosefanajjar** [comment]
@jema28 yeah we need a 404 page as for the SEO this is not something that is caused by our website because I didn't add anything for SEO about Joe or Tempo yet
